docs/getting-started/cli.md

```diff
@@ -0,0 +1,207 @@
+
+To scan your project using the configuration you've set up in Semgrep Cloud Platform **without** sending scan results to Semgrep, use:
+
+```console
+semgrep ci --dry-run
+```
+
+This can be helpful to verify the results of a specific ruleset or to see how your findings change based on the rulesets you choose for your scans.
+
+### Scan using Semgrep's OSS engine
+
+To scan your project with Semgrep's OSS Engine, even though you have the Pro Engine enabled in Semgrep Cloud Platform:
+
+```console
+semgrep ci --oss-only
+```
+
+:::info
+See [Semgrep Pro versus Semgrep OSS](/semgrep-pro-vs-oss) for information on the differences between Semgrep Pro and OSS Engines.
+:::
+
+## Scan using specific Semgrep Products
+
+When you run `semgrep ci`, you scan your project with any product that is enabled in Semgrep Cloud Platform. To scan your project with just one product, run:
+
+```console
+# scan with Semgrep Code
+semgrep ci --code
+
+# scan with Semgrep Supply Chain
+semgrep ci --supply-chain
+
+# scan with Semgrep Secrets
+semgrep ci --secrets
+```
+
+## Extend timeout thresholds
+
+Depending on the file sizes in your project, you may need to increase the timeout threshold so that Semgrep doesn't time out before the scan completes. You can control this value using the `--timeout` flag, which refers to the maximum amount of time Semgrep spends scanning a single file. The default value is 5 seconds. Semgrep attempts to scan each file with this timeout value three times, but you can change this using the `--timeout-threshold` flag:
+
+```console
+# increase timeout to 45 seconds, try only 2 times
+semgrep ci --timeout 30 --timeout-threshold 2
+```
+
+## Improve performance for large codebases
+
+You can set the number of subprocesses Semgrep uses to run checks in parallel:
+
+```console
+semgrep scan -j NUMBER_OF_SUBPROCESSES
+```
+
+By default, the number of jobs Semgrep uses is equivalent to the number of cores detected on the system, but `-j = 1` if you're passing in `--pro`. For additional information, see [Parallelization](/kb/semgrep-code/scan-engine-kill).
+
+## Set log levels
+
+Semgrep provides three levels of logging:
+
+| **Log level** | **Flag** | **Description** |
+| - | - | - |
+| Default | None | Prints scan progress, findings information, warnings, and errors. |
+| Verbose | `-v` or `--verbose` | Includes everything printed when using the default logging level, adding a list of rules and details such as skipped files. |
+| Debug | `--debug` | Logs the entire scan process at a high level of detail. |
+
+### Example usage
+
+To set the logging level for a scan, include the flag when scanning your project:
+
+```console
+# run a scan and get debug logs
+semgrep ci --debug
+```
+
+## Exit codes
+
+The CLI commands `semgrep ci` and `semgrep scan` finish with exit code `0` as long as the scan completes, regardless of whether there were findings. To finish with exit code `1` when there are findings:
```

Comment:
@armchairlinguist Do you mind taking a look at this section again? Not completely sure I explained it correctly (I also added an exit codes section to the OSS doc)

Reply:
Yeah, this isn't quite right yet. The first two are correct, but the the last item is incorrect. You only use `--no-suppress-errors` if you also want internal errors not to be suppressed. To get exit code 1, you only need rules in blocking mode that have findings in the scan.